mdata/store_cassandra_test.go

```diff
@@ -50,13 +50,13 @@ func TestGetTTLTables(t *testing.T) {
 		result := GetTTLTables([]uint32{tc.ttl}, tc.windowFactor, tc.nameFormat)
 		logPrefix := fmt.Sprintf("TTL: %d, WF: %d: ", tc.ttl, tc.windowFactor)
 		if len(result) != 1 {
-			t.Fatalf("%s expected 1 result, got %d", logPrefix, len(result))
+			t.Fatalf("%q expected 1 result, got %d", logPrefix, len(result))
 		}
 		if result[tc.ttl].Table != tc.expectedTableName {
-			t.Fatalf("%s expected table name '%s', got '%s'", logPrefix, tc.expectedTableName, result[tc.ttl].Table)
+			t.Fatalf("%q expected table name %q, got %q", logPrefix, tc.expectedTableName, result[tc.ttl].Table)
 		}
 		if result[tc.ttl].windowSize != tc.expectedWindowSize {
-			t.Fatalf("%s expected window size %d, got %d", logPrefix, tc.expectedWindowSize, result[tc.ttl].windowSize)
+			t.Fatalf("%q expected window size %d, got %d", logPrefix, tc.expectedWindowSize, result[tc.ttl].windowSize)
```

Comment:
%q is only for stuff that needs quoting. the logprefix is not meant to be quoted.